### ISSUE TITLE
Please add a placeholder in the back of "Blocked by".

### ISSUE THREAD

**Issue by lancelot-moon** [AdguardTeam/AdGuardHome#469]
We need to use placeholders in the string #47699751 on OneSky (after the preposition "by") because some languages have a different word order.
Otherwise, its Chinese translation will be strange.

#47699751
Blocked by
-> Blocked by [placeholder]

**Comment by emre1393** [comment]
also Turkish translation needs this, --Blocked by "Filters"-- means --"Filtreler" tarafından engellendi.--
Can you fix this?

**Comment by ameshkov** [comment]
@IldarKamalov plz

**Comment by lancelot-moon** [comment]
Well, you have fixed it as the below.
Thanks.

#47699751
Blocked by
-> Blocked by Filters

**Comment by furunos** [comment]
I would like the following:

```
"blocked_by" : "Blocked by <a href='#filters'>Filters<\/a>"
```

**Comment by ameshkov** [comment]
We're trying to avoid HTML in the strings when it's possible.

**Comment by furunos** [comment]
In order to avoid continuing the word "Filters", I devised it as a way to shorten the display. I understood your policy.

```
Blocked by Filters <a href='#filters'>Filters</a>
```

**Comment by ameshkov** [comment]
Please check how it looks in v0.92 (we'll publish it soon) and re-open this or file a new issue if you think it's not okay.

**Comment by furunos** [comment]
I'm sorry. It seems that it was too early to reflect the result of the translation site to v0.91.
